src/components/AssetsComponent.jsx

```diff
@@ -48,19 +54,55 @@ export class AssetsComponent extends Component {
     }
   }
 
+  addAssetModel = () => {
+  return(<ModelNumberContainer />)
+}
+
   render() {
     return (
       <SideMenuComponent>
         <Header className='landing-heading' content='All Assets' />
         <Table celled>
           <Table.Header>
             <Table.Row>
-              <Table.HeaderCell>Category</Table.HeaderCell>
-              <Table.HeaderCell>Sub-category</Table.HeaderCell>
-              <Table.HeaderCell>Type</Table.HeaderCell>
-              <Table.HeaderCell>Make</Table.HeaderCell>
-              <Table.HeaderCell>Model</Table.HeaderCell>
-              <Table.HeaderCell>Item</Table.HeaderCell>
+              <Table.HeaderCell>
+                  <div className="header">
```

Comment:
The indentation is off from here to end of `render` function. Please fix and ensure you are using indentation of `2 spaces`